seed-server/network-connection.sh

```diff
@@ -111,8 +111,7 @@ topology subnet
 # Additonal optimizations
 txqueuelen 1000
 
-cipher AES-256-CBC
-data-ciphers AES-256-CBC
+data-ciphers AES-256-GCM:AES-128-GCM
```

Comment:
If we change accepted ciphers here,  won't there be a problem during an update, when the server only supports the new ciphers and the client still the old ones.

Reply:
Yes, this is a good point. I will take a look at this and report my findings here. If this is the case then we can temporarily leave the `AES-256-CBC` cipher in the server configuration and remove it in the next release of the VPN components.